.gitlab-ci.yml

```
@@ -212,6 +215,20 @@ run-tests:
     - job: image-build-tests-runner
     - job: image-build-runner
 
+boardfarm-tests:
+  stage: test
+  image: $CI_REGISTRY_IMAGE/prplmesh-boardfarm-ci:$CI_PIPELINE_ID
+  script:
+    - export USER=''
+    - tests/run_bf.sh
+  artifacts:
+    paths:
+      - logs
```

Boardfarm stores it's logs in `results` folder, we should store it also. It have all logs received from consoles of devices during tests and some miscellaneous info.
```suggestion
      - logs
      - results
```

---

Indeed, thanks for the suggestion I applied it.